provisioning/device/src/TransportSettings/ProvisioningClientAmqpSettings.cs

```diff
@@ -36,6 +36,24 @@ public ProvisioningClientAmqpSettings(ProvisioningClientTransportProtocol transp
         /// </remarks>
         public TimeSpan IdleTimeout { get; set; } = TimeSpan.FromMinutes(2);
 
+        /// <summary>
+        /// The client web socket to use when communicating over web sockets.
+        /// </summary>
+        /// <remarks>
+        /// This option is ignored for TCP connections.
+        /// <para>
+        /// If not provided, a client web socket instance will be created for you based on the other
+        /// settings provided in this class.
+        /// </para>
+        /// </remarks>
+        public ClientWebSocket ClientWebSocket { get; set; } = new ClientWebSocket();
```

Comment:
Any migration doc updates?

Reply:
Do we add new features to the migration doc? These didn't exist in v1

Reply:
We call them out, yes.